weblogo/logo_formatter.py

```diff
@@ -327,9 +349,9 @@ def convert(
         self,
         format: str,
         postscript: str,
-        width: int,
-        height: int,
-        resolution: int = 300,
+        width: Optional[float],
```

Comment:
Width, height, and resolution should all be `int` I think.

Reply:
there was a default in logoformat where self.resolution was set to `96.0` so that was throwing off expected types etc. but changing that to `96` allows for changing typehints to `Optional[int]`

Reply:
Good catch. These parameters are passed through to ghostscript, which expects integers.